include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/Problems/IDELinearConstantAnalysis.h

```diff
@@ -42,9 +42,9 @@ class IDELinearConstantAnalysis
     : public IDETabulationProblem<IDELinearConstantAnalysisDomain> {
 private:
   // For debug purpose only
-  static unsigned CurrGenConstantId;
-  static unsigned CurrLCAIDId;
-  static unsigned CurrBinaryId;
+  static unsigned CurrGenConstantId; // NOLINT
```

Comment:
What's the problem here? unused?

Reply:
Core guidelines: avoid non-constant global variables.